### PR TITLE
Tanner Dale

### PR DESCRIPTION
**Pull request by TannerDale**
Here ya go

### CODE REVIEW

File: lib/network.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  def add_show(show)
+    @shows << show
+  end
+
+  def get_all_characters
+    @shows.flat_map do |show|
+      show.characters
+    end
+  end
```
Comment (mikedao) [inline lib/network.rb:17]: Good

Comment (mikedao) [reply lib/network.rb:17]: great pulling out of the logic here

**Review by mikedao — COMMENTED** [review]
Great job throughout.

Score: Pass

**Comment by TannerDale** [comment]
Thank you!

**Pull request closed without merging**